.github/PULL_REQUEST_TEMPLATE/release-checklist.md

```diff
@@ -83,15 +83,22 @@ Check that the release will work:
 - [ ] Update crate versions, commit the changes to the release branch, and do a release dry-run:
 
 ```sh
-cargo release version --verbose --execute --allow-branch '*' --workspace --exclude zebrad --exclude zebra-scan beta
-# Due to a bug in cargo-release, we need to pass an exact version here
-cargo release version --verbose --execute --allow-branch '*' --package zebra-scan 0.1.0-alpha.1
+# Update everything except for alpha crates and zebrad:
+cargo release version --verbose --execute --allow-branch '*' --workspace --exclude zebrad --exclude zebra-scan --exclude zebra-grpc beta
+# Due to a bug in cargo-release, we need to pass exact versions for alpha crates:
```

Comment:
Will be good to add a link to the bug here so we can follow but i was not able to find it.

Reply:
@oxarbitrage do you want to merge the PR as is?

Reply:
Yea, we can do that by now and check in more detail later, i assumed it just don't work so we have to make the workaround but maybe that is not even the case anymore.